DXRModules/DeusEx/Classes/DXRFixup.uc

```diff
@@ -834,3 +834,23 @@ function SetAllLampsState(bool type1, bool type2, bool type3, optional Vector lo
     }
 #endif
 }
+
+function MoveSmugglerElevator()
```

Comment:
This logic could probably go right into the SmugglerElevatorTracker, maybe in PostPostBeginPlay?  Then you never need to even call it, it will just naturally happen because the tracker exists.

Reply:
It near-instantly moves the elevator position. It should only be called when entering a map, not every time an elevator button is pressed.

Reply:
PostPostBeginPlay doesn't run when the trigger is hit.  It just happens as the game is loading a map (or a save)

Reply:
Or are you saying the tracker itself would call it on every entry?